### PR TITLE
Bank massad support

### PR DESCRIPTION
**Pull request by esm7**
Changes:
* Added a Bank Massad scraper.

Bank Massad is basically a subdivision of Beinleumi and its site is completely based on Beinleumi.
The new scraper simply inherits from `BeinleumiScraper` with new login and transaction URLs, while moving these URLs to be class members.
I did not add a test for this scraper because it has nothing to add on top of the Beinleumi test.

### CODE REVIEW

**Comment by esakal** [comment]
Can you create a base class for massad and bein leumi?

Checkout what we have for isracard and amex for example https://github.com/eshaham/israeli-bank-scrapers/blob/849f6d0497cd2829f2b8cb4f289d0f1138b3accc/src/scrapers/base-isracard-amex.ts

The reason behind it is that when changing beinleumi people will not likely to test massad because it is not obvious that one company extend another. using a shared class signal exactly that

**Comment by esm7** [comment]
Done.
My only concern is that this required touching the `BeinleumiScraper` class more than before, and although the changes are seemingly trivial, I don't have a good way to test it.
I could only test the updated `MassadScraper` (now inheriting from `BeinleumiGroupBaseScraper`) which works just like before.

**Comment by esm7** [comment]
Hey there,
When possible, I'd love to get this merged so Bank Massad users will get some scraping going :)

**Comment by esakal** [comment]
@esm7 wow, sorry about that, I forgot to get back to this one. 
I'm approving it. Let's wait few more days for @eshaham to let him provide feedback if any. We will aim to merge it this week.

Thanks again for your contribution

**Review by esakal — APPROVED** [review]

**Comment by esakal** [comment]
@eshaham as we are going to add a new institute please review and confirm. Thanks

**Comment by esakal** [comment]
@eshaham can you check this one please?

File: README.md [hunk 1]
```diff
@@ -266,6 +267,15 @@ const credentials = {
...
 
+## Beinleumi & Massad
+These scrapers are essentially identical and expect the following credentials object:
+```node
+const credentials = {
+  username: <user name>,
+  password: <user password>
+};
+```
+
```
Comment (eshaham) [inline README.md:278]: let's move this up, next to the rest of the banks - I would put it after Mizrahi scraper

**Review by eshaham — APPROVED** [review]
@esm7 thanks for this great contribution 👍 
added a tiny comment, please fix and we can merge 💪

File: README.md (hunk 1 above)
Comment (esm7) [reply README.md]: Done.

**Comment by esakal** [comment]
@esm7 thank you for your contribution!
@eshaham please release new version when you can. In the meanwhile it is deployed as part of the automatic `next` version to npm

**Comment by esakal** [comment]
@eshaham can you please release an official version?

**Pull request merged**